CI/integration_tests/calculators/test_radial_distribution_function.py

```diff
@@ -30,6 +30,7 @@
 from zinchub import DataHub
 
 import mdsuite as mds
+mds.config.memory_fraction = 1
 from mdsuite.utils.testing import assertDeepAlmostEqual
```

Comment:
We have to be careful here, where to set that! If you set it here if will be used in all tests iirc. So it should go in the test definition or in a pytest.fixture.

Reply:
Yes sorry it shouldn't be here at all. This was something I used to check if it was working and not at al meant to be in the PR.

Reply:
In general in the actual running tests I will not have any `scaling=True` unless we want it to be done automatically each time. This is a general thing we can discuss but I am not sure it is necessary.

Reply:
This has been removed.